crates/iota-sdk-ffi/src/types/transaction/mod.rs

```diff
@@ -27,6 +27,50 @@ use crate::{
 
 pub mod v1;
 
+#[derive(Clone, uniffi::Enum)]
+pub enum TransactionDataInner {
+    Version1(Arc<TransactionDataV1>),
```

Comment:
I think I'ld prefer `V1(..)` as the variant name, or did you write it out for a specific reason?